KeenClient/KeenClient.m

```diff
@@ -75,96 +83,6 @@ + (BOOL)validateProjectID:(NSString *)projectID;
  */
 + (BOOL)validateKey:(NSString *)key;
 
-/**
- Returns the path to the app's library/cache directory.
- @returns An NSString* that is a path to the app's documents directory.
- */
-- (NSString *)cacheDirectory;
-
-/**
- Returns the root keen directory where collection sub-directories exist.
- @returns An NSString* that is a path to the keen root directory.
- */
-- (NSString *)keenDirectory;
-
-/**
- Returns the direct child sub-directories of the root keen directory.
- @returns An NSArray* of NSStrings* that are names of sub-directories.
- */
-- (NSArray *)keenSubDirectories;
-
-/**
- Returns all the files and directories that are children of the argument path.
- @param path An NSString* that's a fully qualified path to a directory on the file system.
- @returns An NSArray* of NSStrings* that are names of sub-files or directories.
- */
-- (NSArray *)contentsAtPath:(NSString *)path;
-
-/**
- Returns the directory for a particular collection where events exist.
- @param collection The collection.
- @returns An NSString* that is a path to the collection directory.
- */
-- (NSString *)eventDirectoryForCollection:(NSString *)collection;
```

Comment:
Thanks for getting rid of this and pathForEventInCollection()! My favorite way to up CC numbers is to delete stuff :)